src/script/E2EIdentity/Modals/Modals.ts

```diff
@@ -224,6 +224,7 @@ export const getModalOptions = ({
           action: secondaryActionFn,
           text: t('acme.done.button.secondary'),
         },
+        ...hideCloseBtn,
```

Comment:
this removes the close button and revent closing the modal by clicking outside it
```
const hideCloseBtn = {hideCloseBtn: true, preventClose: true};
```